Base/Python/slicer/util.py

```diff
@@ -1372,10 +1372,12 @@ def reloadScriptedModule(moduleName):
     for item in items:
         parent.layout().removeItem(item)
 
-    # create new widget inside existing parent
-    widget = eval("reloaded_module.%s(parent)" % widgetName)
```

Comment:
If we remove the use of the `reloaded_module` ivar, we should also revisit 

https://github.com/Slicer/Slicer/blob/14a4056f2bc935b7bd54e5eaa13941aa4c60aa66/Base/Python/slicer/util.py#L1346-L1356

Reply:
I agree with @jcfr - this code never seemed like a clean implementation but as I recall it was the best I could figure out at the time.  If there's a cleaner and more maintainable implementation it would be great.

Reply:
Current the imp.load_module call still seems to be required. Without it, the module is not actually reloaded.